src/lib/jsonParseV2/parseExpression.h

```diff
@@ -1,9 +1,9 @@
-#ifndef SRC_LIB_JSONPARSEV2_PARSESCOPEVECTOR_H_
```

Comment:
It's funny... github is detecting this as a rename in a file, while parseExpression.h y totally new :)

NTC (informative)